atlasexec/copilot.go

```diff
@@ -6,7 +6,11 @@ import (
 )
 
 type (
-	CopilotParams struct{ Prompt, Session string }
+	CopilotParams struct {
+		Prompt, Session string
+		// Write and Delete glob patterns to specify file permissions.
+		Write, Delete string
```

Comment:
We usually try to name the parameter as close as possible to the real ones
```suggestion
		FSWrite, FSDelete string
```

Reply:
Atlas currently only has FS permission flags, this is why the atlas flag looks like this: `fs.write` but they can also mean something like `db.` (I guess this is what Ariel had in mind). Therefore Write and Delete are good imho. cc @a8m